pkg/deploy/controller/deployer/metrics.go

```diff
@@ -0,0 +1,34 @@
+package deployment
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// DeploymentConfigControllerSubsystem is how this controller is represented in
+// prometheus metrics.
+const DeployerControllerSubsystem = "deployer_controller"
+
+var (
+	deployerPodErrorsCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Subsystem: DeployerControllerSubsystem,
+			Name:      "failure_count",
+			Help:      "Counter that counts total number of deployer pod errors per error type",
```

Comment:
@kargakis i don't think prometheus has that, you can query that via prometheus console